### ISSUE TITLE
Greyed out download button on OpenNeuro website

### ISSUE THREAD

**Issue by botsunny** [connectomicslab/Aneurysm_Detection#5]
Hi there! I'm not sure if it's just me but the Download tab is greyed out on the OpenNeuro website. What is the recommended way to download this dataset?

Thanks!

![image](https://github.com/connectomicslab/Aneurysm_Detection/assets/71499037/ec86e5ec-9a66-45d1-800f-d45a9d0d57bf)

**Comment by tommydino93** [comment]
Hi @botsunny,
Thanks for your interest here. Weird that the button is grayed out. It looks fine on my side.

![image](https://github.com/connectomicslab/Aneurysm_Detection/assets/37182324/449e276f-2aae-41f4-bd60-8cc5be2f935d)

Can you please check with one of your colleagues if it's also grayed out for them?

**Comment by botsunny** [comment]
Hi @tommydino93,
It seems like it's an issue exclusive to my PC, but I've been able to download the dataset using [openneuro-py](https://pypi.org/project/openneuro-py/). Thanks!

However, out of curiosity, what does this particular size refer to?

![image](https://github.com/connectomicslab/Aneurysm_Detection/assets/71499037/955d948b-142d-48ec-acb7-282bb41339e2)

As after downloading the dataset I realised it is not just 13.46 GB but approximately 200. Thanks again!
